src/sql/parts/notebook/notebookInput.ts

```diff
@@ -208,12 +263,13 @@ export class NotebookInput extends EditorInput {
 		return this._parentContainer;
 	}
 
-	/**
-	 * An editor that is dirty will be asked to be saved once it closes.
-	 */
-	isDirty(): boolean {
-		return this._model.isDirty;
-	}
+	// /**
+	//  * An editor that is dirty will be asked to be saved once it closes.
+	//  */
+	// isDirty(): boolean {
```

Comment:
Are we bringing this back later? Or is it just that this isn't needed as the parent class already checks the model? I don't understand why this is removed.

Reply:
I don's think we need 'dirty' getter and setters in NotebookInput going forward. It's got dependcy with notebook model save implementation. Need to clean exisitng save code once this feature is fully tested.